plugins/logs/mask.go

```diff
@@ -153,6 +153,15 @@ func (r maskRule) Mask(event *EventV1) error {
 		}
 		maskObj = event.Result
 		maskObjPtr = &event.Result
+	case partNDBCache:
```

Comment:
Because of the careful interface/type-wrangling we do earlier in the server's `Log` function, we can just copy/paste the masking logic here for the NDBCache. It behaves *exactly* like the `Input` and `Result` masking targets, so the logic is safe to reuse with minimal changes. :smile: